src/stream/src/executor/project/materialized_exprs.rs

```diff
@@ -194,7 +231,17 @@ impl<S: StateStore> Inner<S> {
                     let post_commit = self.state_table.inner.commit(barrier.epoch).await?;
                     let update_vnode_bitmap = barrier.as_update_vnode_bitmap(self.actor_ctx.id);
                     yield Message::Barrier(barrier);
-                    post_commit.post_yield_barrier(update_vnode_bitmap).await?;
+
+                    // evict LRU cache
+                    self.state_table.cache.evict();
+
+                    if let Some((_, cache_may_stale)) =
```

Comment:
So the `must_use` lint doesn't work here?

Reply:
md